airflow/providers/amazon/aws/config_templates/config.yml

```diff
@@ -0,0 +1,131 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Config for the ecs executor, using the provider package mechanism added by @potiuk (if you could review this piece specifically and ensure it's done correctly, I'd appreciate that)

Reply:
Isn't this meant to be [here](https://github.com/apache/airflow/blob/8ecd576de1043dbea40e5e16b5dc34859cc41725/airflow/providers/amazon/provider.yaml#L717) instead?

Reply:
I guess maybe?   I can move it.

Reply:
Config has been moved